src/SystemAbstractions/Abstractions/IMicrosoft.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.SystemAbstractions
+{
+    public interface IMicrosoft
```

Comment:
>IMicrosoft [](start = 21, length = 10)

With this change, can we also remove the Shim for Microsoft.Win32.Registry from mscorlib.fakes?

Reply:
Done.